src/Kris/LaravelFormBuilder/Events/BeforeFormValidation.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace Kris\LaravelFormBuilder\Events;
```

Comment:
I think it shouldn't be called multiple times, there is no really a reason for that.
Some ppl may use this event to redirect back, or to log something, who knows.

It can be added around this line https://github.com/kristijanhusak/laravel-form-builder/blob/master/src/Kris/LaravelFormBuilder/Form.php#L1069.
Maybe something like this:

```php
$isValid = !$this->validator->fails();

$this->fire(new AfterFormValidation($isValid, $form, $this->validator));

return $isValid;
```